src/main/java/io/debezium/connector/yugabytedb/connection/OpId.java

```diff
@@ -65,6 +66,26 @@ private static byte[] parseKey(String keyString) {
         return Base64.getDecoder().decode(keyString);
     }
 
+    public void setTerm(long term) {
```

Comment:
In my opinion, setters are not needed, creating a new object and assigning should do the same job. Refer to my other comment in `YugabyteDBSnapshotChangeEventSource`